### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by kryvets-vadym**
[DEMO LINK](https://kryvets-vadym.github.io/react_static-list-of-todos/)

### CODE REVIEW

File: src/components/TodoList/TodoList.tsx [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map(todo => {
+        return (
```
Comment (bohdan-stepanets) [inline src/components/TodoList/TodoList.tsx:14]: No need in return if it is Arrow function

File: src/components/TodoInfo/TodoInfo.tsx [hunk 2]
```diff
@@ -1,4 +1,37 @@
...
+
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { user } = todo;
```
Comment (bohdan-stepanets) [inline src/components/TodoInfo/TodoInfo.tsx:12]: You make double `{ smth } = todo`

File: src/components/TodoInfo/TodoInfo.tsx [hunk 3]
```diff
@@ -1,4 +1,37 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
```
Comment (bohdan-stepanets) [inline src/components/TodoInfo/TodoInfo.tsx:1]: Will be cool if you move some css properties from main App.scss to Smth.scss

**Review by bohdan-stepanets — APPROVED** [review]
Good job but read please comments in TodoInfo.tsx and TodoList.tsx

File: src/components/TodoInfo/TodoInfo.tsx (hunk 3 above)
Comment (Olexandr6) [reply src/components/TodoInfo/TodoInfo.tsx:1]: +1

File: src/components/TodoList/TodoList.tsx (hunk 1 above)
Comment (Olexandr6) [reply src/components/TodoList/TodoList.tsx:14]: +1

**Review by Olexandr6 — CHANGES_REQUESTED** [review]
Fix comments that you colleague mentioned above

File: src/components/TodoInfo/TodoInfo.tsx (hunk 3 above)
Comment (Olexandr6) [reply src/components/TodoInfo/TodoInfo.tsx:1]: Let's move styles from App.scss to separate files as well

**Review by Olexandr6 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**